packages/common-scripts/src/common.ts

```diff
@@ -9,7 +9,9 @@ import secp256k1Blake160Multisig from "./secp256k1_blake160_multisig";
 import { FromInfo, parseFromInfo } from "./from_info";
 import secp256k1Blake160 from "./secp256k1_blake160";
 import { getConfig, Config } from "@ckb-lumos/config-manager";
-import locktimePool from "./locktime_pool";
+import locktimePool, {
+  CellCollector as locktimePoolCellCollector,
```

Comment:
```suggestion
  CellCollector as LocktimePoolCellCollector,
```

Reply:
already have `LocktimePoolCellCollector`  in file, Can I change to `LocktimePoolCellCollectorClass`?

Reply:
maybe we can change it to `LocktimePollCellCollectorConstructor` ?